.github/labels.yml

```
@@ -66,6 +66,14 @@
     color: '#91e4e5'
     aliases: [type-shell-xonsh]
 
+# OS
+  - name: os::cygwin
+    description: pertains to Cygwin, commonly run on Windows
+    color: '#d0e0e3'
+  - name: os::msys2
+    description: pertains to MSYS2, commonly run on Windows
```

Should we explicitly specify call out git-bash to help remember?

```suggestion
    description: pertains to MSYS2 and Git for Windows
```

---

Yeah, that's a good idea. Thanks!